frontend/model/contracts/group.js

```diff
@@ -1373,6 +1378,11 @@ sbp('chelonia/defineContract', {
       await sbp('gi.db/archive/save', key, proposals)
       sbp('okTurtles.events/emit', PROPOSAL_ARCHIVED, contractID, proposalHash, proposal)
     },
+    'gi.contracts/group/removeArchivedProposals': async function (contractID) {
+      const { username } = sbp('state/vuex/state').loggedIn
+      const key = `proposals/${username}/${contractID}`
+      await sbp('gi.db/archive/delete', key)
+    },
```

Comment:
Please move this down below next to `'gi.contracts/group/removeArchivedPayments'` so that the file stays nicely organized